deps/spidershim/test/value.cc

```diff
@@ -1434,3 +1434,177 @@ TEST(SpiderShim, ArrayBuffer_NeuteringApi) {
   Local<Float64Array> f64a =
       CreateAndCheck<Float64Array, 8>(buffer, 8, 127);
 }
+
+static inline v8::Local<v8::Script> v8_compile(v8::Local<v8::String> x) {
+  v8::Local<v8::Script> result;
+  if (v8::Script::Compile(v8::Isolate::GetCurrent()->GetCurrentContext(), x)
+          .ToLocal(&result)) {
+    return result;
+  }
+  return v8::Local<v8::Script>();
+}
+
+static inline v8::Local<v8::Value> CompileRun(v8::Local<v8::String> source) {
```

Comment:
Well, for now the tests cannot be imported verbatim anyways, and given that our `V8Engine` setup is different than `cctest`'s, I'd rather keep the tests calling the `V8Engine` `CompileRun` method.  FWIW this is fixed on master.